src/app/(with-nav-footer)/(mypage)/my-activities/add-activities/_components/ImageUploader.tsx

```diff
@@ -0,0 +1,104 @@
+import { toast } from 'react-toastify';
+import { useRef } from 'react';
+import { useUploadActivityImage } from '@/lib/hooks/useActivities';
+import Image from 'next/image';
+
+interface ImageUploaderProps {
+  value: string | string[];
+  onChange: (val: string | string[]) => void;
+  single?: boolean;
+  multiple?: boolean;
+  limit?: number;
+}
+
+export default function ImageUploader({
+  value,
+  onChange,
+  single = false,
+  multiple = false,
+  limit,
+}: ImageUploaderProps) {
+  const inputRef = useRef<HTMLInputElement>(null);
+  const { mutateAsync: uploadImage } = useUploadActivityImage();
+
+  const handleClick = () => {
+    inputRef.current?.click();
+  };
+
+  const handleChange = async (e: React.ChangeEvent<HTMLInputElement>) => {
+    const file = e.target.files?.[0];
+    if (!file) return;
+
+    if (!['image/jpeg', 'image/png', 'image/webp'].includes(file.type) || file.size > 5 * 1024 * 1024) {
+      toast.error('지원되지 않는 형식이거나 5MB를 초과했습니다.');
+      return;
+    }
+
+    const formData = new FormData();
+    formData.append('image', file);
+
+    try {
+      const { activityImageUrl } = await uploadImage(formData);
+      if (single) {
+        onChange(activityImageUrl);
+      } else {
+        const updated = Array.isArray(value) ? [...value, activityImageUrl] : [activityImageUrl];
+        onChange(updated);
+      }
+    } catch (err) {
+      console.log(err);
+      toast.error('이미지 업로드에 실패했습니다.');
+    }
+  };
+
+  const handleRemove = (index: number) => {
+    if (single) {
+      onChange('');
+      return;
+    }
+
+    if (!Array.isArray(value)) return;
+
+    const updated = [...value];
+    updated.splice(index, 1);
+    onChange(updated);
+  };
+
+  const images = Array.isArray(value) ? value : value ? [value] : [];
+
+  return (
+    <div className='mt-[8px] flex flex-wrap gap-4'>
+      {images.map((img, idx) => (
+        <div key={idx} className='relative h-32 w-32'>
+          <Image src={img} className='h-full w-full rounded object-cover' alt='preview' />
```

Comment:
```suggestion
          <Image src={img} width={180} height={180} className='h-full w-full rounded object-cover' alt='preview' />
```
너비와 높이를 지정해주지 않으니, 사진을 등록하려하면 오류가 발생하네요! 이 부분 추가해주시면 될 것 같아용